File: 812. Largest Triangle Area/solution.py
class Solution(object):
  def largestTriangleArea(self, points):
    """
    :type points: List[List[int]]
    :rtype: float
    """
    result = 0
    for i in points:
      for j in points:
        for k in points:
          area = 0.5 * abs(i[0] * j[1] + j[0] * k[1] + k[0] * i[1]- j[0] * i[1] - k[0] * j[1] - i[0] * k[1])
          if area > result:
            result = area
    return result

File: 812. Largest Triangle Area/solution.js
/**
 * @param {number[][]} points
 * @return {number}
 */
 var largestTriangleArea = function(points) {
  let result = 0;
  for (let i of points){
    for (let j of points){
      for (let k of points){
        let area = 0.5 * Math.abs(i[0] * j[1] + j[0] * k[1] + k[0] * i[1]- j[0] * i[1] - k[0] * j[1] - i[0] * k[1]);
        if (area > result){
          result = area;
        }
      }
    }
  }
  return result;
};